recipe/meta.yaml

```diff
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
```

Comment:
Is there a compelling reason to separate out the compiled libraries into their own package `libavahi` and then have the `avahi` package that includes the scripts, so that users of the library don't have to install the scripts?

This is done already for a number of other libraries (`{lib}curl` being a good example) and follows the standard runtime package setup for RHEL/Debian.

Reply:
If I understand correctly, should I rename the output to `libavahi`?

Reply:
I think we should leave this conversation for later, since we require this binaries to compile Qt

Reply:
Sorry for another delay, I agree, let's do this later, it was just an idea (rather than a request).